manim/utils/opengl.py

```diff
@@ -31,11 +31,11 @@ def orthographic_projection_matrix(
         return projection_matrix
 
 
-def perspective_projection_matrix(width=None, height=None, near=4, far=18, format=True):
+def perspective_projection_matrix(width=None, height=None, near=2, far=30, format=True):
     if width is None:
-        width = config["frame_width"] / 3
+        width = config["frame_width"] / 6
     if height is None:
-        height = config["frame_height"] / 3
+        height = config["frame_height"] / 6
```

Comment:
Can you add a comment on how you got `6` and what happens changing from `3` to `6`?

Reply:
It was actually just trying different numbers until something looked about right. It wouldn't make much sense to add a comment about what these values do here as it would involve a lot of math about projection matrices. https://jsantell.com/3d-projection/